modules/looker-core/README.md

```diff
@@ -0,0 +1,193 @@
+# Looker Core module
+
+This module manages the creation of a [Looker Core instance](https://cloud.google.com/looker/docs/looker-core).
+
+This module accepts Oauth client ID and secret in the input variable `oauth_config` in case you have
+already [set up an oauth client and credentials](https://cloud.google.com/looker/docs/looker-core-create-oauth).
+If that is not the case it is possible to specify support_email in the same variable `oauth_config` for a default oauth
+client id and secret setup within the terraform script, be aware that **such an oauth client id is not suitable for
+authenticating end users**, and it is only used to provision the looker core instance.
+You'll still be forced to create a new oauth and update the looker core instance from the console (or gcloud) as there
+is no terraform support for these resources.
+
+
+> [!WARNING]
+> Please be aware that, at the time of this writing, deleting the looker core instance via terraform is not possible due
+> to https://github.com/hashicorp/terraform-provider-google/issues/19467. The WA is to delete the instance from the
+> console (or gcloud with force option) and remove the corresponding resource from the terraform state.
+
+<!-- TOC -->
+
+* [Looker Core module](#looker-core-module)
+    * [Examples](#examples)
+        * [Simple example](#simple-example)
+        * [Looker Core private instance with PSA](#looker-core-private-instance-with-psa)
+        * [Looker Core full example](#looker-core-full-example)
+    * [Variables](#variables)
+    * [Outputs](#outputs)
+
+<!-- TOC -->
+
+## Examples
+
+### Simple example
+
+This example shows how to set up a public Looker Core instance.
+
+```hcl
+module "looker" {
+  source     = "./fabric/modules/looker-core"
+  project_id = var.project_id
+  region     = var.region
+  name       = "looker"
+  network_config = {
+    public = {}
+  }
+  oauth_config = {
+    support_email = "support@google.com"
+  }
+}
+# tftest modules=1 resources=3 inventory=simple.yaml e2e
```

Comment:
E2E tests fail due to not active IAP service. Can you add it, and also any other required services [here](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/tests/examples_e2e/setup_module/main.tf#L24-L57) ?